setup.py

```diff
@@ -102,7 +102,7 @@ def packages():
     },
     cmdclass={'test': PyTest},
     tests_require=tests_runtime_require + [
-        'flake8>=3.5.0',
+        'flake8>=3.7.7',
```

Comment:
Version 3.7.0 added native support for `per-file-ignores`.
http://flake8.pycqa.org/en/latest/user/options.html#cmdoption-flake8-per-file-ignores